src/content/1.7/development/webservice/_index.md

```diff
@@ -61,16 +61,16 @@ Each access key is defined by this information:
 Now that your access key is generated you can test your store's webservice, its endpoint is located in the `/api/` folder at the root of your installation of Prestashop.
 The quickest way to test your API is to use your browser:
 
-* If PrestaShop is installed at the root of your server, you can access the API here: http://example.com/api/
-* If PrestaShop is installed in a subfolder of your server, you can access the API here: http://example.com/prestashop/api/
+* If PrestaShop is installed at the root of your server, you can access the API here: https://example.com/api/
+* If PrestaShop is installed in a subfolder of your server, you can access the API here: https://example.com/prestashop/api/
```

Comment:
As these examples are used for development, it is likely that someone working on its webservice is using localhost and consequently does not have https enabled. Maybe `https` is not relevant here ?